profile/rwahs.xml

```diff
@@ -8063,6 +8066,13 @@
       <typeRestrictions>
         <restriction type="library"/>
       </typeRestrictions>
+      <groupAccess>
```

Comment:
This is copied from [`library_object_ui`](https://github.com/kehh/installation-profile/blob/9c15cd22a68ee49d4e5d43d10deae7045426412e/profile/rwahs.xml#L5358-L5364) using the updated `derive-displays.xslt` below: